.setup/install_system.sh

```diff
@@ -232,6 +232,7 @@ apt-get install -qqy imagemagick
 
 apt-get -qqy autoremove
 
+apt-get install libssl-dev
```

Comment:
What does adding this library do/support?

Also, it needs `-qqy` flag (or at least `-y`) to work properly.